Assets/ScreenCapture.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ScreenCapture : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.S))
+            CaptureScreen();
+    }
+
+    public void CaptureScreen()
+    {
+        UnityEngine.ScreenCapture.CaptureScreenshot("test.png");
```

Comment:
Maybe we could do something like this to hide the UI screen when taking a screenshot.
```
public void OnClickScreenCaptureButton()
 {
    StartCoroutine(CaptureScreen());
 }
 public IEnumerator CaptureScreen()
 {
    // Wait till the last possible moment before screen rendering to hide the UI
    yield return null;
    GameObject.Find("MyCanvas").GetComponent<Canvas>().enabled = false;
 
    // Wait for screen rendering to complete
    yield return new WaitForEndOfFrame();
 
    // Take screenshot
    Application.CaptureScreenshot("screenshot.png");
 
    // Show UI after we're done
    GameObject.Find("MyCanvas").GetComponent<Canvas>().enabled = true;
 }
```